src/components/call-tool.js

```diff
@@ -77,9 +77,36 @@ module.exports = React.createClass({
     );
   },
   renderForm: function() {
+    // We need to ensure there is enough space in the
+    // input to show the placeholder in ultiple languages.
     var placeholder = this.context.intl.formatMessage({id: 'enter_phone'});
-    if (!this.state.number || this.state.number !== "(+" + this.state.countryPrefix + ") ") {
-      placeholder = "";
+    var countryPrefix = "(+" + this.state.countryPrefix + ")";
+
+    // Default placeholder state is a fresh page load state.
+    // Example: "(+44) placeholder text"
+    var placeholderContainer = (
+      <span className="placeholder-container">
+        <span className="placeholder-width">{countryPrefix}&nbsp;</span>
+        <span className="placeholder">{placeholder}&nbsp;</span>
+      </span>
+    );
+
+    // This state is for when the input has been cleared,
+    // completely empty, so no county prefix.
+    if (!this.state.number) {
+      placeholderContainer = (
+        <span className="placeholder-container">
+          <span className="placeholder-width">{placeholder}&nbsp;</span>
+        </span>
+      );
+    // This state is for when the input is not empty and not default.
```

Comment:
The comment placement here is a bit confusing, would this work for you?
```
}
// This state is for [...]
else if (....) {
  ..
}
```

Reply:
That works for me.